lib/orbf/rules_engine/services/ethiopian_calendar.rb

```diff
@@ -0,0 +1,61 @@
+# the code from #from_iso and #to_iso is a mix of https://github.com/gyohannes/ethiopic_date
+# and where we drop custom code to use what Date can offer to manipulate
+# Julian day number https://en.wikipedia.org/wiki/Julian_day
```

Comment:
🤯

The `from_iso` takes a gregorian date and returns an Ethiopian, while the `to_iso` takes an Ethiopian date and returns a Gregorian. And we, Western world, are using the Gregorian. Right?

Reply:
yes